cmd/earthly/root_cmds.go

```diff
@@ -311,6 +311,30 @@ Set up a whole custom git repository for a server called example.com, using a si
 				},
 			},
 		},
+		{
+			Name:        "web-ui",
+			Description: "Access the web UI",
+			Usage:       "Access the web UI via your default browser and print the url",
+			UsageText:   "earthly web-ui (--provider=github|--final=ci)",
+			Hidden:      true,
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:        "provider",
```

Comment:
Is this flag even needed? I imagine that the user would be able to just select in the UI itself the form of login, right?

Reply:
If this flag is provided, and the provider is valid, we attempt to log the user in automatically - without the need to click a button.
If / when we add more providers this will also work with those.

As-is, I can run
```
earthly web-ui --provider=github
```
to open the web ui and attempt a login via GitHub. 

If / when we add more providers we can simply use:
```
earthly web-ui --provider=bitbucket
```
without zero changes to the CLI.

Reply:
Ok sounds good

Reply:
Ok sounds good